### ISSUE TITLE
Change variant_ID 

### ISSUE THREAD

**Issue by CPJVink** [woocommerce/facebook-for-woocommerce#2136]
In our business it is essential to post photos of multiple products and then tag the FB products. Facebook allows to tag product by either the name, ID of the product or by content_ID.  

Currently, the content_ids are constructed by SKU ID of WooCommerce + Variant ID of WooCommerce, eg. 56790_164278. Unfortunately, the variant ID of a product is automatically constructed by WooCommerce. It is, therefore, impossible for us to tag product because we do know the SKU ID's but we do not know every variant ID. Due to the high number of products it would be inefficient to create a csv mapping all the variant IDs. 

Before this plugin, we had an external program that created a google sheet that sync with the FB shop and there we set content_id equal to SKU_ID + size_of_a_product which allowed for tagging products with ease. Hopefully, there is a way to edit the code such that we can set the content_id with our own logic, which will then enable us to tag product with ease. 

In other words: Is there a way to change the assigned content_id by a custom field, because we do know how to create unique numbers from our data eg. SKU ID + size_of_a_product. 

For completeness, it is not practically to tag products by product names because these are not unique  and ID of the product is randomly generated. Therefore, since content_IDs are created with logic, we hoped to customize this logic. 

Thank you in advanced and hoping to hear from you.

**Comment by rawdreeg** [comment]
This issue has been resolved by #2169, which provides a filter to modify the retailer ID. I will close this.

Thanks
